code/campaign/build.gradle

```diff
@@ -237,17 +237,30 @@ task platformFunctionalTestJacocoReport(type: JacocoReport, dependsOn: "createPh
 
 apply from: 'checkstyle.gradle'
 
+configurations.configureEach {
+    resolutionStrategy.cacheChangingModulesFor 0, 'seconds'
+}
+
 dependencies {
     // adobe sdk dependencies
-    implementation "com.adobe.marketing.mobile:core:$rootProject.mavenCoreVersion"
-    testImplementation "com.adobe.marketing.mobile:core:$rootProject.mavenCoreVersion"
+    // implementation "com.adobe.marketing.mobile:core:$rootProject.mavenCoreVersion"
+    // testImplementation "com.adobe.marketing.mobile:core:$rootProject.mavenCoreVersion"
+    implementation ("com.github.adobe.aepsdk-core-android:core:feature~uiservice-SNAPSHOT") {}
```

Comment:
we should make a note to update this to the release version when available

Reply:
Will create a Jira ticket with release readiness tasks